### PR TITLE
Merge 4.8.1 into 4.9.0

### PR DESCRIPTION
**Pull request by sebasfalcone**
|Related issue|
|---|
| #24415 |

### CODE REVIEW

**Comment by sebasfalcone** [comment]
## Failing tests
- Framework tests are falling due to an update of one of the dependencies
  - Tracked by: https://github.com/wazuh/wazuh/issues/24506
  
- Vulnerability scanner QA test is failing due to a misalignment in the generated content
  - Tracked by: https://github.com/wazuh/intelligence-platform/issues/1619

**Review by Damian-Mangold — APPROVED** [review]

File: framework/requirements.txt [hunk 1]
```diff
@@ -43,6 +43,7 @@ idna==3.7.0
 importlib-metadata==6.8.0
 inflection==0.3.1
 isodate==0.6.1
+itsdangerous==2.0.0
```
Comment (Selutario) [inline framework/requirements.txt:46]:
```suggestion
```

I think this dep should not be included :thinking: It was removed in 4.9.0 when `connexion` was upgraded to 3.0.0:
- https://github.com/wazuh/wazuh/pull/23199/files#diff-ac0d503454a85f64c8021ded91db3e9afddcc51dd88864ca250ac0e199da797a 

Also, it is not being included in the deps 30 cpython:
image

**Review by Selutario — CHANGES_REQUESTED** [review]

**Review by Selutario — APPROVED** [review]

**Comment by JcabreraC** [comment]
- **macOS compilation tests / build-sonoma (pull_request)**
Expected, fixed in 4.9.0: https://github.com/wazuh/wazuh/pull/23927

**Comment by rauldpm** [comment]
Service tests are using old packages due to the wazuh-packages migration to wazuh/wazuh, pending removal

**Review by JcabreraC — APPROVED** [review]

**Pull request merged**